src/templates/landingv2.tsx

```
@@ -0,0 +1,45 @@
+"use client"
```

ESLint

[error] Strings must use singlequote.

(style/quotes)